pm/sendermonitor.go

```diff
@@ -180,6 +184,19 @@ func (sm *senderMonitor) QueueTicket(addr ethcommon.Address, ticket *SignedTicke
 	sm.senders[addr].queue.Add(ticket)
 }
 
+// IsLocked checks whether a sender's unlock period ends the round after the next round
+func (sm *senderMonitor) IsLocked(addr ethcommon.Address) error {
```

Comment:
This method name is a misnomer - the sender isn't actually locked if the method returns an error. What about calling this method `ValidateSender()` (duplicate method names in this type and in `Recipient` wouldn't be a problem if we choose not to modify the `Recipient` public API)?